HousingSearchApi/V1/Interfaces/AssetQueryGenerator.cs

```diff
@@ -16,18 +19,35 @@ public AssetQueryGenerator(IWildCardAppenderAndPrepender wildCardAppenderAndPrep
 
         public QueryContainer Create(HousingSearchRequest request, QueryContainerDescriptor<QueryableAsset> q)
         {
-            if (string.IsNullOrWhiteSpace(request.SearchText)) return null;
+            if (!(request is HousingSearchRequest housingSearchRequest)) return null;
```

Comment:
Given that the `request` input parameter is strongly typed on the method definition, this check will never fail and so could be removed.

Reply:
This PR needs tests - both E2E and unit.